rich-message-plugin/CustomEvents_AddButtons_Stylesheet.css

```diff
@@ -141,7 +141,11 @@ p {
 
 .embeddedServiceSidebar.layout-docked
   .embeddedServiceLiveAgentStateChatItem.agent
-  .chatContent {
+  .chatContent
+  ,
+.embeddedServiceSidebar.layout-docked
+  .embeddedServiceLiveAgentStateChatItem.agent
```

Comment:
I didn’t get the empty line on line 145 and formatting on line 146

Reply:
Updated